adapters/index.go

```diff
@@ -49,7 +49,13 @@ func (a *IndexAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pb
 	if req.App != nil {
 		return nil, fmt.Errorf("Index doesn't support apps")
 	}
-	indexReq := makeOpenRTBGeneric(req, bidder, a.FamilyName())
+	mediaTypes := []pbs.MediaType{pbs.MEDIA_TYPE_BANNER, pbs.MEDIA_TYPE_VIDEO}
```

Comment:
How did you decide which adapters can support both media types? Did you confirm with tests or you just assuming the endpoint supports it? I noticed Rubicon didn't have video supported but others did...